### PR TITLE
Update LDAPMonitorDataSource.py

### PR DESCRIPTION
**Pull request by linkslice**
password needs to be single quoted in case there's special chars in it.  e.g. $

### CODE REVIEW

**Comment by jregovic** [comment]
This is still outstanding and needs a merge. I just experienced this problem today.

**Comment by pcarinhas** [comment]
Is there a Jira for this defect?

File: ZenPacks/zenoss/LDAPMonitor/datasources/LDAPMonitorDataSource.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def getCommand(self, context):
         if self.ldapBindDN:
             parts.append('-D "%s"' % self.ldapBindDN)
         if self.ldapBindPassword:
-            parts.append('-P "%s"' % self.ldapBindPassword)
+            parts.append('-P \'%s\'' % self.ldapBindPassword)
         if self.ldapBindVersion:
```
Comment (pcarinhas) [inline ZenPacks/zenoss/LDAPMonitor/datasources/LDAPMonitorDataSource.py]: Looks fine.

**Comment by csmouse** [comment]
PR has aged out. Closing.

**Pull request closed without merging**